cmd/up/space/init.go

```diff
@@ -110,7 +116,10 @@ type initCmd struct {
 func init() {
 	// NOTE(tnthornton) we override the runtime.ErrorHandlers so that Helm
 	// doesn't leak Println logs.
-	runtime.ErrorHandlers = []func(error){} //nolint:reassign
+	kruntime.ErrorHandlers = []func(error){} //nolint:reassign
+
+	spacesScheme = runtime.NewScheme()
+	upboundv1alpha1.AddToScheme(spacesScheme)
```

Comment:
No need for the scheme. Just add it ot the client-go one. For clients that's totally fine and we do that in many places:
```suggestion
	kruntime.Must(upboundv1alpha1.AddToScheme(spacesScheme))
```
Then also the helper below is unnecessary.